Automattic/wp-calypso
# Add Fiverr card to home task

Related to https://github.com/Automattic/dotcom-forge/issues/1652

## Proposed Changes

* Add Fiverr home task card

![Fiverr](https://user-images.githubusercontent.com/6586048/220650018-45b945f8-7e4e-40aa-9b37-c81dc5b05583.png)


## Testing Instructions



* Apply this diff: D102369-code
* Go to /home/:site
* See if the task card shows up

## Pre-merge Checklist
